src/container/RelayRenderer.js

```diff
@@ -342,12 +281,53 @@ class RelayRenderer extends React.Component {
     this.mounted = false;
   }
 
+  /**
+   * @private
+   */
+  _resolveContainerProps(): Object {
```

Comment:
Agree, but I added `type RelayContainerProps = {[propName: string]: mixed}` for it, because it needs to be changed in multiple places in this file.